docs/nsm_api.md

```diff
@@ -359,16 +364,17 @@ NSE is the actual provider of a network service, to make aware NSM of the servic
 
 ```proto
 message EndpointAdvertiseRequest {
-   repeated netmesh.NetworkServiceEndpoint network\_endpoint = 1;
+    string request_id = 1;
+    netmesh.NetworkServiceEndpoint network_endpoint = 2;
 }
 
 message NetworkServiceEndpoint {
-   string network_service_name = 1;
-   string network_service_host = 2;
-   string nse_provider_name = 3;
-   string nse_provider_namesapce = 4;
-   string socket_location = 5;
-   repeated common.Interface interface = 6;
+    string network_service_name = 1;
```

Comment:
We should have labels here as well (simple map<string,string> labels)

Reply:
I can add it, but could you clarify its purpose so it can be named accordingly.